guides/doc-Managing_Hosts/topics/Registering_Hosts.adoc

```diff
@@ -22,7 +22,7 @@ You can also register Atomic Hosts to {ProjectServer} or {SmartProxyServer}.
 
 Use one of the following procedures to register a host:
 
-* xref:registering-a-host-to-project-using-the-global-registration-template_managing-hosts[]
+* xref:registering-a-host_managing-hosts[]
```

Comment:
Interesting, this renders correctly actually, but the ID does not exist.

https://docs.theforeman.org/nightly/Managing_Hosts/index-satellite.html

TIL this.